implementations/micrometer-registry-otlp/src/test/java/io/micrometer/registry/otlp/OtlpConfigTest.java

```diff
@@ -276,4 +277,25 @@ void histogramPreferenceUseEnvVarWhenConfigNotSet() throws Exception {
                 .isEqualTo(HistogramFlavor.BASE2_EXPONENTIAL_BUCKET_HISTOGRAM));
     }
 
+    @Test
+    void histogramFlavorPerMeter() {
+        Map<String, String> properties = new HashMap<>();
+        properties.put("otlp.histogramFlavorPerMeter",
+                "a.b.c=explicit_bucket_histogram ,expo =base2_exponential_bucket_histogram");
+        OtlpConfig otlpConfig = properties::get;
+        assertThat(otlpConfig.validate().isValid()).isTrue();
+        assertThat(otlpConfig.histogramFlavorPerMeter()).containsExactly(
+                entry("a.b.c", HistogramFlavor.EXPLICIT_BUCKET_HISTOGRAM),
+                entry("expo", HistogramFlavor.BASE2_EXPONENTIAL_BUCKET_HISTOGRAM));
+    }
+
+    @Test
+    void maxBucketsPerMeter() {
+        Map<String, String> properties = new HashMap<>();
+        properties.put("otlp.maxBucketsPerMeter", "a.b.c = 10");
```

Comment:
Is spacing intentional?
```suggestion
        properties.put("otlp.maxBucketsPerMeter", "a.b.c=10");
```